.github/workflows/reviewdog.yml

```
@@ -24,6 +24,23 @@ jobs:
           fail_on_error: true
           pattern: TODO_IN_THIS_
 
+  # Ensure that we are using polylog (via logger.) instead of the golang's default
+  # log package.
+  check_polylog_in_off_chain_package:
+    name: Check polylog in off-chain source code
```

Is there any reason to limit this to off-chain code?

---

NIT:

```suggestion
  # Ensure that we are using polylog (via logger.) instead of the golang's stdlib
  # log package.
  check_stdlog_in_off_chain_package:
    name: Check stdlog in off-chain source code
```

---

@bryanchriswhite I committed your suggestion, but the reason it's limited to `off-chain` is because my goal (with this change) was to prevent someone accidentally using the std logger in our off-chain packages. Some of the default scaffolded work still uses `log.`. I haven't gone through it's uses and simply considered it out of scope.

Do you want to add a TODO or something else?